Example/DocumentScannerApp/ViewController.swift

```diff
@@ -110,6 +112,14 @@ extension ViewController {
 
         present(cropViewController, animated: true)
     }
+
+    @IBAction func actionShare(_ sender: UIButton) {
+        guard let image = self.scannedImage
+            else { return }
+
+        let vc = UIActivityViewController(activityItems: [image], applicationActivities: [])
+        present(vc, animated: true, completion: nil)
```

Comment:
There is no need to explicitly set the completion to `nil`. So I recommend to leave it out, like this: 

```suggestion
        present(vc, animated: true)
```